test/server/services/user.js

```diff
@@ -203,5 +216,22 @@ describe('User service tests', () => {
           done();
         }, done);
     });
+    it('should promote user to admin, deactivate, and ban user by ID', (done) => {
+      User.updateUserById(this.userManageParams1)
+        .then((user) => {
+          expect(user.attributes.is_admin).to.equal(true);
```

Comment:
Can we verify that other properties are unaffected? If not, there's some Bookshelf magic we need to do in the .save() invocation in the model to not clear out fields that are not explicitly set in the update.

Reply:
I think I can verify it. We can make it a more encompassing updateUserById service that includes all of the current information that is untouched. Very much like the user edit page.

Reply:
This is done in tests below.